app/scripts/modules/kubernetes/src/v2/manifest/ManifestKindSearch.tsx

```diff
@@ -0,0 +1,29 @@
+import { IPromise } from 'angular';
+
+import { SearchService } from '@spinnaker/core';
+
+interface IManifestKindSearchResults {
+  account: string;
+  kubernetesKind: string;
+  name: string;
+  namespace: string;
+  provider: string;
+  region: string;
+  // spinnakerKind
```

Comment:
Oh that's meant to indicate that `type` (below) is really the `spinnakerKind`. But it would be more clear if the comment was to the right of the field.